tests/test_pfc.py

```diff
@@ -0,0 +1,101 @@
+import time
+from swsscommon import swsscommon
+
+
+def getBitMaskStr(bits):
+
+    mask = 0
+
+    for b in bits:
+        mask = mask | 1 << b
+
+    return str(mask)
+
+
+def setPortPfc(dvs, port_name, pfc_queues):
+
+    cfg_db = swsscommon.DBConnector(swsscommon.CONFIG_DB, dvs.redis_sock, 0)
+
+    port_qos_tbl = swsscommon.Table(cfg_db, 'PORT_QOS_MAP', '|')
+    fvs = swsscommon.FieldValuePairs([('pfc_enable', ",".join(str(q) for q in pfc_queues))])
+    port_qos_tbl.set(port_name, fvs)
+
+    time.sleep(1)
+
+
+def setPortPfcAsym(dvs, port_name, pfc_asym):
+
+    cfg_db = swsscommon.DBConnector(swsscommon.CONFIG_DB, dvs.redis_sock, 0)
+
+    port_tbl = swsscommon.Table(cfg_db, 'PORT', '|')
+    fvs = swsscommon.FieldValuePairs([('pfc_asym', pfc_asym)])
```

Comment:
I am a little confused here. setPortPfcAsym align with doporttask in portsorch which subscribes app db instead of config db from my understanding. How come it can subscribe change in config db?

Reply:
That is correct, portsorch subscribes for a port table in APP_DB.
Portsyncd subscribes for a port table in CFG_DB, handles config and applies it to APP_DB.
So then portsorch got a notification and processes new config values.